packages/shared/src/components/post/common/SharedPostLink.tsx

```diff
@@ -1,33 +1,35 @@
 import React, { ReactElement, ReactNode } from 'react';
 import Link from '../../utilities/Link';
-import { Post } from '../../../graphql/posts';
+import { Post, SharedPost } from '../../../graphql/posts';
 import { CombinedClicks } from '../../../lib/click';
 
 interface SharedPostTitleProps {
-  post: Post;
+  sharedPost: SharedPost;
+  mainSource: Post['source'];
   children: ReactNode;
   className?: string;
   onGoToLinkProps?: CombinedClicks;
 }
 
 export const SharedPostLink = ({
-  post,
+  sharedPost,
+  mainSource,
```

Comment:
We don't have any variable named `source` here and we can distinguish between the main and the shared because we have to use `sharedPost.source` for the second one. 
I'm trying to change this just to stay consistent through the codebase ✔️

Reply:
source and sharedPostSource would be better then wouldn't you agree?